src/lib/unpackPlotProps.js

```diff
@@ -28,10 +121,9 @@ export default function unpackPlotProps(props, context) {
     multiValued = true;
   }
 
-  let isVisible = false;
-  if (props.show || (fullValue !== void 0 && fullValue !== null)) {
-    isVisible = true;
-  }
+  const isVisible = isValidCustomConfigObject(customConfig)
```

Comment:
here I would use a rule like `isVisible = permittedByCustomRules() && (hasFullValue() || props.show)` (assuming that you remove the `hasFullValue` checked from inside the `pemitted` function

Reply:
disregarding the props.show is what was making the dropzone dissapear, the logic above restores it